### PR TITLE
fix: REPLAT-8454 fix slow scrolling on magazine sections

### PR DESCRIPTION
**Pull request by jmars**
Magazine sections have one slice with 30 tiles, this means the flatlist loading on slices doesn't work and we get really bad performance.

### CODE REVIEW

**Comment by infreezer** [comment]
Do you see the same error? 
I'm using iPad - medium breakpoint. 
It's interesting the error isn't there in wide breakpoint.
image

**Comment by jmars** [comment]
Flex doesn't work inside flatlists, I think switching at 10 items is probably a reasonable compromise? Less than 10 won't have scrolling issues and it's unlikely to see a vertical layout with 10 items on tablet?

**Comment by infreezer** [comment]
> Flex doesn't work inside flatlists, I think switching at 10 items is probably a reasonable compromise? Less than 10 won't have scrolling issues and it's unlikely to see a vertical layout with 10 items on tablet?

What do you think about having two components?
And use the component with the flastlist where it's needed in this case [here](https://github.com/newsuk/times-components/blob/master/packages/edition-slices/src/slices/standard/index.js).
instead of having "if" in the vertical layout.

**Comment by jmars** [comment]
Makes sense, I haven't done much slice work so hadnt thought of it, will make the changes

File: packages/edition-slices/src/slices/standard/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Standard extends Component {
   }
 
   renderSmall(breakpoint) {
-    return <VerticalLayout tiles={this.getTiles(breakpoint)} />;
+    return <ListVerticalLayout tiles={this.getTiles(breakpoint)} />;
```
Comment (infreezer) [inline packages/edition-slices/src/slices/standard/index.js]: What is the scope of the story?
Do we need to use the ListVerticalLayout for tablet also?
Please check [this line.](https://github.com/newsuk/times-components/blob/38be789d92bf9ede7040d828cd71f977801c62ba/packages/edition-slices/src/slices/standard/index.js#L36)

**Comment by infreezer** [comment]
Do we need storybook integration of the new component?

File: packages/slice-layout/__tests__/list.base.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        expect(wrapper).toMatchSnapshot();
+      }
+    },
+    {
+      name: "two child elements",
+      test() {
+        const wrapper = renderComponent(
+          <ListVerticalLayout
+            tiles={[
+              <test tile={{ article: { id: 'testId'}}} />,
```
Comment (ValGeorgiev) [inline packages/slice-layout/__tests__/list.base.js]: do we need this <test /> component, if yes, the test name it doesn't look appropriate

**Comment by jmars** [comment]
I looked at the other slices and the other uses seem to have a fixed number of items in vertical layout? That is fine, its just the huge list we get on mobile magazine sections that doesnt work well, and that is a variable number.

**Pull request merged**